src/components/JoinOurCommunity.tsx

```diff
@@ -0,0 +1,45 @@
+import DiscordLogo from "../assets/discord_logo.png";
+import SlackLogo from "../assets/slack_logo.png";
+
+export default function JoinOurCommunity() {
+    return (
+        <>
+            <h1 className="text-zinc-300 text-center mb-4 text-4xl font-extrabold leading-none tracking-tight md:text-5xl mt-36 lg:text-8xl">
+                Join Us
+            </h1>
+            <p className="text-zinc-300 text-center text-lg font-normal lg:text-2xl px-10 sm:px-16 xl:px-96 mt-10">
+                Community Description
+            </p>
```

Comment:
Why is this paragraph tag needed?

Reply:
Description/brief about community can be added if there is any, I will remove it if its not needed.

Reply:
@harshkhandeparkar  is this needed?

Reply:
I don't think it is necessary but we can write a one liner like Join for future updates, queries and to make connections...